src/ocm/components/clusterConfiguration/StorageDetail.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import { TextContent, Text, TextVariants, Grid, GridItem } from '@patternfly/react-core';
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  TableVariant,
+  RowWrapperProps,
+  RowWrapper,
+  IRow,
+} from '@patternfly/react-table';
+import { ExtraParamsType } from '@patternfly/react-table/dist/js/components/Table/base';
+import {
+  Disk,
+  fileSize,
+  getInventory,
+  Host,
+  ValidationInfoActionProps,
+  WithTestID,
+} from '../../../common';
+import DiskRole, { onDiskRoleType } from '../../../common/components/hosts/DiskRole';
+import DiskLimitations from '../../../common/components/hosts/DiskLimitations';
+
+type StorageDetailProps = {
+  canEditDisks?: (host: Host) => boolean;
+  onDiskRole?: onDiskRoleType;
+  host: Host;
+  AdditionalNTPSourcesDialogToggleComponent?: ValidationInfoActionProps['AdditionalNTPSourcesDialogToggleComponent'];
+  hideNTPStatus?: boolean;
+};
+
+type SectionTitleProps = {
+  title: string;
+};
+
+type DisksTableProps = {
```

Comment:
And here we can also update the prop types.
```suggestion
type DisksTableProps = {
  canEditDisks: (host: Host) => boolean;
  onDiskRole: onDiskRoleType;
  host: Host;
  disks: Disk[];
  installationDiskId?: string;
};
```